### PR TITLE
#1840 Docs: read version from gradle.properties

### PR DESCRIPTION
**Pull request by vitojeng**
Sample file preview: https://vitojeng-ohara.readthedocs.io/en/docs/sample.html

required by #1840 

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/conf.py [hunk 1]
```diff
@@ -183,10 +226,12 @@
...
 
-
 # extlinks
 #   https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html#module-sphinx.ext.extlinks
 
-extlinks = {}
+extlinks = {
+    'issue': ('https://github.com/oharastream/ohara/issues/%s', ''),
+    'source': ('https://github.com/oharastream/ohara/blob/%s/' % ohara_branch + "%s", ''),
+    'kafka': ('https://issues.apache.org/jira/browse/KAFKA-%s', '')
```
Comment (chia7712) [inline docs/src/conf.py]: 這是用來？

File: docs/src/sample.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+A sampe file for rst practice
+==============================
+
+- Ohara version |version|
+- Github issue link: :issue:`issue <800>`
+- Github source: :source:`PageViewRegionExample <ohara-streams/src/test/java/com/island/ohara/streams/examples/PageViewRegionExample.java>`
```
Comment (chia7712) [inline docs/src/sample.rst]: 嗯？

File: docs/src/conf.py (hunk 1 above)
Comment (vitojeng) [reply docs/src/conf.py]: extlinks 是 sphinx 的一個 plugin, 可以用來自訂一些常用的連結. 方便寫 rst 文件時使用.
目前我先加入三個用來測試. 至於測試內容請見 sample.rst

File: docs/src/sample.rst (hunk 2 above)
Comment (vitojeng) [reply docs/src/sample.rst]: 這只是用來測試, 順便給大家看一下, 可以先讓大家 preview.
不過目前 readthedocs 的 build 出現問題(ModuleNotFoundError), 還需要做設定解決.

File: docs/src/conf.py (hunk 1 above)
Comment (chia7712) [reply docs/src/conf.py]: 如果是這樣的話 可以把docker k8s zookeeper也加上去

Comment (chia7712) [reply docs/src/conf.py]: 如果是這樣的話 可以把docker k8s zookeeper也加上去

Comment (chia7712) [reply docs/src/conf.py]: 如果是這樣的話 可以把docker k8s zookeeper也加上去

Comment (vitojeng) [reply docs/src/conf.py]: 透過讀取 gradle.properties 的 version, 再搭配使用 extlinks,
我想或許可以提供 ohara 文件撰寫在連結部份的需要.

Comment (vitojeng) [reply docs/src/conf.py]: > 如果是這樣的話 可以把docker k8s zookeeper也加上去

Ok. 請問 docker 的 issue 連結是?

File: docs/src/sample.rst (hunk 2 above)
Comment (vitojeng) [reply docs/src/sample.rst]: Preview: https://vitojeng-ohara.readthedocs.io/en/docs/sample.html

File: docs/src/conf.py (hunk 1 above)
Comment (vitojeng) [reply docs/src/conf.py]: k8s, zookeeper 加上去了. docker 目前看不出要加那個, 以後再加...

File: docs/src/sample.rst (hunk 2 above)
Comment (vitojeng) [reply docs/src/sample.rst]: 先改用 Python 2.7 解決

**Pull request merged**